Formula/gdbm.rb

```diff
@@ -19,6 +19,7 @@ def install
     args = %W[
       --disable-dependency-tracking
       --disable-silent-rules
+      --without-readline
```

Comment:
I'll add a comment, but there is no link for the issue because I reported it via email.